Source/Core/VideoCommon/TextureUtils.h

```diff
@@ -4,13 +4,26 @@
 #pragma once
 
 #include <string>
+#include <unordered_set>
 
 #include "Common/CommonTypes.h"
 
 class AbstractTexture;
 
 namespace VideoCommon::TextureUtils
 {
+class TextureDumper
+{
+public:
+  // Only dumps if texture did not already exist anywhere within the dump-textures path.
+  void DumpTexture(const ::AbstractTexture& texture, std::string basename, u32 level,
+                   bool is_arbitrary);
+
+private:
+  std::unordered_set<std::string> m_dumped_textures;
+};
+
 void DumpTexture(const ::AbstractTexture& texture, std::string basename, u32 level,
```

Comment:
NBD but wondering if it'd be clearer if we change this to something else like `ForceDumpTexture`?

Reply:
oh, I meant to keep the `File::Exists` check to retain the existing behavior, and that would make `Force` a weird name.

Reply:
Ah ok.  I'm fine with either.